src/rocm_docs/__init__.py

```diff
@@ -33,9 +33,9 @@ def __init__(
         self.extensions: List[str] = []
         self.html_title: str
         self.html_theme_options: Dict[str, Union[str, bool, List[str]]] = {}
-        self.doxygen_root: MaybePath
-        self.doxygen_project: Tuple[Optional[str], MaybePath]
-        self.doxyfile: MaybePath
+        self.doxygen_root: MaybePath = None
+        self.doxygen_project: Tuple[Optional[str], MaybePath] = (None, None)
```

Comment:
```suggestion
        self.doxygen_project: Dict[str, Union[MaybePath, Optional[str]]] = {}
```
This type declaration is inconsistent with the value set on line 55.

Reply:
GitHub double-included this for some reason :/